services/src/main/java/org/exoplatform/webconferencing/listeners/CallRecordingNotificationListener.java

```diff
@@ -42,19 +42,23 @@ public void onEvent(Event<CallInfo, Map<? extends String, ? extends String>> eve
         Map<? extends String, ? extends String> info = event.getData();
         List<String> participants = callInfo.getParticipants().stream().map(UserInfo::getId).collect(Collectors.toList());
         String status = info.get("status");
-        String uploadingUser = info.get("user_id");
         String fileName = info.get("file_name") == null ? "" : info.get("file_name");
         String identity =  info.get("identity") == null ? null : info.get("identity");
         String callType = info.get("type") == null ? "" : info.get("type");
         WebConferencingService webConferencingService = ExoContainerContext.getCurrentContainer().getComponentInstanceOfType(WebConferencingService.class);
+        Map<String, String> identityInfo = webConferencingService.getNameAndAvatarByIdentityAndType(callType, identity);
+        String name = identityInfo.get("name") == null ? "" : identityInfo.get("name");
+        String avatarUrl = identityInfo.get("avatar_url") == null ? "" : identityInfo.get("avatar_url");
```

Comment:
No need to make the getNameAndAvatar part in webconferencing service. 
You can make it all in this function it will be clearer

Reply:
In webconferencing, we have code related to webconferencing. Here you need data from spaceservice and identityManager. So call them directly, as you call webconferencing service